client/accept-invite/logged-out-invite/signup-form.jsx

```diff
@@ -103,6 +98,15 @@ export default React.createClass( {
 		)
 	},
 
+	footerLink() {
+		let logInUrl = config( 'login_url' );
+		if ( config.isEnabled( 'login' ) ) {
+			logInUrl = '/log-in';
+		}
+		logInUrl += '?redirect_to=' + encodeURIComponent( window.location.href );
+		return <a href={ logInUrl } className="logged-out-form__link">{ this.translate( 'Already have a WordPress.com account? Log in now.' ) }</a>;
```

Comment:
I don't believe that we're properly component-izing the CSS here. If I recall, this is how it's done in the signup framework as well.

We should fix how we handle this, although it could be in a different PR.